### DISCUSSION TITLE
Local flood data

### DISCUSSION THREAD

**Discussion by mattlucht** [digital-land/data-standards-backlog#58] [Planning consideration]
We're interested in exploring the various types of flood data that may be held by local planning authorities and other organisations such as the Environment Agency

**Comment by RichLimbrick** [comment]
We @ Camden have locally defined flood risk areas - and these are treated as a planning constraint as well as a trigger for requiring a flood risk assessment (FRA). - so it's also linked to L[ocal Application Requirements](https://github.com/digital-land/data-standards-backlog/discussions/52). Because we define them by street, these are currently plotted as lines in the middle of the street, so we are using buffers to create a polygon that can pick up property boundaries. This is not totally accurate though (false positives and false negatives, depending on buffer distance) so may need to plot the polygons. Would be interested to know how other local authorities treat and record more localised flood data.

**Reply by mattlucht** [reply]
that's really helpful, thanks! Locally defined flood risk areas feel like an interesting topic to dive into. I'll propose something with the LPA working group and see whether we can get a discussion going on this! I know that @greg-slater his some interest in this area too!

**Reply by kieranwint** [reply]
Hi @RichLimbrick,
Thanks again for this comment, we've been doing some digging on local flood data and previously flooded streets this week as a result!
We found out some good information from looking at your Local Plan! Also within this, it says LFRZs are defined as discrete areas of flooding that do not exceed the national criteria for a ‘Flood Risk Area’, and are wondering:
a) where exactly is this defined?
b) what is the national criteria which LFRZs fall short of? Is it that these areas just fall short of being designated within flood risk zone 2?
Thanks in advance!
